spec/features/course/unread_status_management_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe 'Announcements read/unread status management', type: :feature do
+  subject { page }
+
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+
+    let(:user) { create(:user, role: :administrator) }
+    let(:course) { create(:course) }
+
+    before do
+      login_as(user, scope: :user)
+    end
+
+    context 'with a new announcement' do
+      let!(:announcement) { create(:course_announcement, course: course) }
+
+      it 'unread number increases by 1' do
```

Comment:
See all the <code>it</code> examples here: http://betterspecs.org

normally like <code> it is/has/does </code>